.circleci/config.yml

```diff
@@ -210,6 +210,9 @@ jobs:
                     if [[ $(cat $FNAME | grep -x ".*datasets.*limo.*" | wc -l) -gt 0 ]]; then
                       python -c "import mne; print(mne.datasets.limo.data_path(subject=1, update_path=True))";
                     fi;
+                    if [[ $(cat $FNAME | grep -x ".*datasets.*megref_noise.*" | wc -l) -gt 0 ]]; then
+                      python -c "import mne; print(mne.datasets.megref_noise.data_path(update_path=True))";
```

Comment:
These two lines must be wrong because CircleCI has the download status:

https://20202-1301584-gh.circle-artifacts.com/0/dev/auto_examples/preprocessing/plot_find_ref_artifacts.html#sphx-glr-auto-examples-preprocessing-plot-find-ref-artifacts-py

Looks like megref_noise -> refmeg_noise

Can you check the output and see if it's otherwise how you want it to look?

The topomaps look a bit crazy but that's a separate plotting bug I think (feel free to open an issue), not something you changed here

Reply:
The first plot now looks more cramped than it did before, but I could live with it. Otherwise it seems fine.

I noticed the topomaps as well. My guess is that this is somehow related to throwing out every other channel to reduce the size of the dataset.